react/src/components/ButtonLink.js

```diff
@@ -1,14 +1,11 @@
-import { Component } from 'react';
 import { Link } from 'react-router-dom';
 
-class Button extends Component {
-  render() {
-    return (
-      <Link {...this.props} className="btn">
-        {this.props.children}
-      </Link>
-    );
-  }
+function Button(props) {
```

Comment:
The properties that should be relevant within this component (and for Link) are `to` and `children`. These can be destructured from props and used as needed.

Reply:
addressed https://github.com/sentry-demos/empower/pull/368/commits/4a22bbfd9a744a0523ca742b11a4895df889764c